src/base/kaldi-utils.h

```diff
@@ -46,10 +46,14 @@
 #endif
 #endif
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER)
 #  define KALDI_MEMALIGN(align, size, pp_orig) \
   (*(pp_orig) = _aligned_malloc(size, align))
 #  define KALDI_MEMALIGN_FREE(x) _aligned_free(x)
+#elif defined(__CYGWIN__)
+#  define KALDI_MEMALIGN(align, size, pp_orig) \
+  (*(pp_orig) = aligned_alloc(align, size))
```

Comment:
Actually I now see that we don't need this macro at all any more, as aligned_alloc is part of the C++11 standard.  So please remove the macro and replace all instances of
  `foo = KALDI_MEMALIGN(x, y, &z)`
with
  `foo = aligned_alloc(x, y);`
and remove the temporary variables that previously bound to z, and replace KALDI_MEMALIGN_FREE with free (there are instances of this in the cudamatrix/ directory as well, that should also be replaced).